eip/EIPS/eip-4337.md

```diff
@@ -624,6 +624,219 @@ Returns an array of the entryPoint addresses supported by the client. The first
 }
 ```
 
+#### * eth_chainId
+
+Returns EIP155 Chain ID.
+
+```json=
+# Request
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "eth_chainId",
+  "params": []
+}
+
+# Response
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": "0x1"
+}
+```
+
+### debug Namespace
+
+#### * debug_bundler_clearState
+
+Clears the bundler mempool and reputation data of paymasters/accounts/factories/aggregators.
+
+```json=
+# Request
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "debug_bundler_clearState",
+  "params": []
+}
+
+# Response
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": "ok"
+}
+```
+
+#### * debug_bundler_dumpMempool
+
+Returns:
+
+`array` - Array of mempool entries objects, or an empty array.
+
+where each mempool entry has the following fields:
+
+`userOp` - UserOperation object. See `eth_sendUserOperation` for more details.
+`prefund` - The calculated prefund required for the user operation execution.
+`aggregator` - (optional) the aggregator used by the senderaccount.
+
+```json=
+# Request
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "debug_bundler_dumpMempool",
+  "params": []
+}
+
+# Response
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": [
+    {
+      "userOp": {
+        sender, // address
+        nonce, // uint256
+        initCode, // bytes
+        callData, // bytes
+        callGasLimit, // uint256
+        verificationGasLimit, // uint256
+        preVerificationGas, // uint256
+        maxFeePerGas, // uint256
+        maxPriorityFeePerGas, // uint256
+        paymasterAndData, // bytes
+        signature // bytes
+      },
+      "prefund": "0x1234abcd",
+      "aggregator": "0x7A0A0d159218E6a2f407B99173A2b12A6DDfC2a6"
+    }
+  ]
+}
+```
+
+#### * debug_bundler_sendBundleNow
+
+Forces the bundler to execute the entire current mempool as `handleOps()` transaction.
+
+```json=
+# Request
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "debug_bundler_sendBundleNow",
+  "params": []
+}
+
+# Response
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": "ok"
+}
+```
+
+#### * debug_bundler_setBundleInterval
+
+Sets bundling mode and max mempool size.
+parameters:
+
+`interval` - number | 'manual' | 'auto'
+`maxPoolSize` - number
+
+```json=
+# Request
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "debug_bundler_setBundleInterval",
+  "params": ["manual", 100]
+}
+
+# Response
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": "ok"
+}
+```
+
+#### * debug_bundler_setReputation
+
+Sets reputation of given addresses. parameters:
+
+
+`array` - Aray of reputation objects, each with the following fields:
+
+`address` - The address to set the reputation for.
+
+`opsSeen` - number of user operations seen linked to this address.
+
+`opsIncluded` - number of user operations included  in a mined bundle linked to this address.
```

Comment:
number of userOps linked to this address that were included in past bundles (by any bundler)